resources/uco_design_document.md

```diff
@@ -71,24 +71,56 @@ To achieve its targeted objectives UCO has chosen to pursue an ontological appro
 # 3 Objects
 
 UCO conceptual content is represented as a graph with nodes and edges.
-The nodes in the graph are objects (instances of ontology classes) and the edges are explicitly defined relationships between objects.
+The nodes in the graph are objects (individual instances of ontology classes) and the edges are explicitly defined relationships between objects.
 
-It is important to note the difference in granularity between the RDF graph and the UCO domain graph.
-In the RDF graph ALL instances of classes and all literal values of properties are objects/nodes and all properties are edges.
+In an RDFS/OWL ontology all classes represent a scope/set of individual instances of particular "concepts". Each individual (object in the graph) has an ID and is asserted as a member of particular classes via type statements. 
+
+To express simple literal characteristic properties of an individual member of a class we utilize datatype properties. Datatype properties are asserted as predicates of RDF triples (subject predicate object) where the individual is the subject and a literal is the object. For example, a concept class "Car" may have a simple datatype property "color" and an RDF triple for a given individual instance of the Car class could be expressed as [car-42 color "green"].
+
+To express characteristic properties of an individual member of a class that are more complicated than simple literals (typically made up of multiple aspects/properties) then we need to utilize object properties. Object properties are asserted as predicates of RDF triples where the individual is the subject and an individual object instance of a class is the object. For example, a concept class "Person" may have an object property "hasChild" and an RDF triple for a given individual instance of the Person class could be expressed as [person-4 hasChild person-12].
```

Comment:
I wish to highlight for those on the Facet Design Pattern Working-Group the wording here:

> To express characteristic properties of an individual member of a class that are more complicated than simple literals (typically made up of multiple aspects/properties) then we need to utilize object properties. Object properties are asserted as predicates of RDF triples where the individual is the subject and an individual object instance of a class is the object.

This wording defining "object property" doesn't seem to be exactly aligned with the definition of an `ObjectProperty` in OWL.  It seems a little broader to me, and lends support to `Facet`s, which inhabit a design space of not being domain objects, kind of not being objects, and kind of being property-sets.